services/visual-studio-marketplace/visual-studio-marketplace-downloads.service.js

```diff
@@ -13,7 +13,7 @@ module.exports = class VisualStudioMarketplaceDownloads extends VisualStudioMark
     return {
       base: '',
       pattern:
-        '(visual-studio-marketplace|vscode-marketplace)/:measure(d|i)/:extensionId',
+        '(visual-studio-marketplace|vscode-marketplace)/:platform(ado)?/:measure(d|i)/:extensionId',
```

Comment:
I elected to use the shorthand `ado` here instead of the lengthier `azure-devops`. However, given the convenience of the new modal window around populating badge URLs, there may not be much value in the shorthand. 

Let me know if you think the longer version would be better